tests/Doctrine/Tests/Common/Collections/ClosureExpressionVisitorTest.php

```diff
@@ -267,6 +363,20 @@ public function testSortByFieldDescending(): void
         self::assertEquals('a', $objects[2]->getFoo());
     }
 
+    public function testSortByFieldKeepOrderWhenSameValue(): void
+    {
+        $firstElement  = new TestObject('a');
+        $secondElement = new TestObject('a');
+
+        $objects = [$firstElement, $secondElement];
+        $sort    = ClosureExpressionVisitor::sortByField('foo');
+
+        usort($objects, $sort);
+
+        self::assertSame($firstElement, $objects[0]);
+        self::assertSame($secondElement, $objects[1]);
```

Comment:
You can compare with an expected array here. Like
```suggestion
        self::assertSame([$firstElement, $secondElement], $objects);
```

Reply:
Done.